chrome_launcher.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Download Extension
+extension_setup() {
+  ps_analysis_tool_version=v0.3.0
+  cd /tmp/
+  if [ ! -d /tmp/ps-analysis-tool-$ps_analysis_tool_version ]; then
+    mkdir -p /tmp/ps-analysis-tool-$ps_analysis_tool_version
+    curl -L -O -s  https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/download/$ps_analysis_tool_version/extension-$ps_analysis_tool_version.zip
+    unzip -qo extension-$ps_analysis_tool_version.zip -d /tmp/ps-analysis-tool-$ps_analysis_tool_version
+  fi
+}
+
+# Detect OS
+if [[ "$(uname)" == "Darwin" ]]; then
+  echo "Macintosh"
+  launch_chrome() {
+      local CHROME_PATH="/Applications/Google Chrome.app"
+      local DATA_DIR="$(mktemp -d '/tmp/chrome_data_dir.XXXXXXXXXX')"
+      open -W -na "${CHROME_PATH}" --args \
+          --disable-sync \
+          --no-default-browser-check \
+          --no-first-run \
+          --user-data-dir="${DATA_DIR}" \
+          "$@" https://example.com >/dev/null 2>&1 && rm -rf "${DATA_DIR}" &
+  }
+elif [[ "$(uname)" == "Linux" ]]; then
+  echo "Linux"
+  launch_chrome() {
+    local CHROME="google-chrome"
+    local DATA_DIR="$(mktemp -d '/tmp/chrome_data_dir.XXXXXXXXXX')"
+    "${CHROME}" \
+        --disable-sync \
+        --no-default-browser-check \
+        --no-first-run \
+        --user-data-dir="${DATA_DIR}" \
+        "$@" https://example.com >/dev/null 2>&1 && rm -rf "${DATA_DIR}" &
+    }
+fi
+
+# Loads Chrome with a temporary profile that is deleted after Chrome is closed
+chrome-default() {
+  launch_chrome \
+    --install-autogenerated-theme='150,220,150'
+}
+
+chrome-3pcd() {
```

Comment:
@MiteshShah can you please add one liner before each chrome function for it's purpose? e.g. line number [41](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/211/files#diff-89ef72b5c5b4c0decec968e6b38afaffaaabecc1db74269094fd8b0f65f84c51R41)